content/en/docs/faq/troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Troubleshooting"
+linkTitle: "Troubleshooting"
+weight: 60
+type: "docs"
+---
+
+When troubleshooting cert-manager your best friend is `kubectl describe`, this will give you information on the resources as well as recent events. It is not advised to use the logs as these are quite verbose and only should be looked at if the following steps do not provide help.
+
+cert-manager consists of multiple custom resources that live inside your Kubernetes cluster, these resources are linked together and are often created by one another. When such an event happens it will reflect in a Kubernetes event, you can see these per namespace using `kubectl get event`, in case of getting them of a specific resource `kubectl describe` will help you a lot.
```

Comment:
```suggestion
cert-manager consists of multiple custom resources that live inside your Kubernetes cluster, these resources are linked together and are often created by one another. When such an event happens it will be reflected in a Kubernetes event, you can see these per-namespace using `kubectl get event`, or in the output of `kubectl describe` when looking at a single resource.
```